INSTALL.md

```diff
@@ -81,28 +83,22 @@ for more details.
 Choice of Build and Installation Procedure
 ------------------------------------------
 
-There are two partially overlapping infrastructures available to build
-Coq. They are available through `Makefile.make` (legacy / hybrid
-build) and `Makefile.dune` (full Dune build).
-
-You can use the `COQ_USE_DUNE` environment variable to change the one
-to use by default. This is useful for Coq development, where we
-recommend to rely mainly on `Makefile.dune`. Note that mixing the two
-systems is not perfectly supported and may lead to confusing behavior.
+Coq is normally built with `dune`, but a legacy/hybrid build is also
+available via `Makefile.make`.
```

Comment:
I think that the previous paragraph was more informative, and as mentioned by Gaëtan, the legacy / hybrid build is still the recommended method for users / packages for now (while the pure Dune build is recommended for devs).